src/NLog.Azure.Kusto.Tests/ADXSinkE2ETest.cs

```diff
@@ -13,25 +13,21 @@ public class ADXSinkE2ETest : IDisposable
     {
         private readonly string? m_generatedTableName;
         private readonly KustoConnectionStringBuilder? m_kustoConnectionStringBuilder;
-        private readonly KustoConnectionStringBuilder? m_kustoConnectionStringBuilderDM;
 
         public ADXSinkE2ETest()
         {
-            Assert.NotNull(Environment.GetEnvironmentVariable("INGEST_ENDPOINT") ?? throw new ArgumentNullException("INGEST_ENDPOINT not set"));
+            Assert.NotNull(Environment.GetEnvironmentVariable("CONNECTIONSTRING") ?? throw new ArgumentNullException("CONNECTIONSTRING not set"));
             Assert.NotNull(Environment.GetEnvironmentVariable("DATABASE") ?? throw new ArgumentNullException("DATABASE name not set"));
             Assert.NotNull(Environment.GetEnvironmentVariable("APP_ID") ?? throw new ArgumentNullException("APP_ID not set"));
             Assert.NotNull(Environment.GetEnvironmentVariable("AZURE_TENANT_ID") ?? throw new ArgumentNullException("AZURE_TENANT_ID not set"));
             Assert.NotNull(Environment.GetEnvironmentVariable("APP_KEY") ?? throw new ArgumentNullException("APP_KEY not set"));
```

Comment:
Test case needs to be fixed.  SInce it is all done via connection string, app id and other credentials will not be take from env.

Please take reference from https://github.com/Azure/azure-kusto-nlog-sink/blob/feature/ConnectionStringAuth/src/NLog.Azure.Kusto.Tests/ADXSinkE2ETest.cs


```suggestion
            KustoConnectionStringBuilder.DefaultPreventAccessToLocalSecretsViaKeywords = false;
            string dmConnectionStringEndpoint = connectionString.Contains("ingest-") ? connectionString : connectionString.ReplaceFirstOccurrence("://", "://ingest-");
            string engineConnectionStringEndpoint = !connectionString.Contains("ingest-") ? connectionString : connectionString.ReplaceFirstOccurrence("ingest-", "");```